### PR TITLE
Release Nodes 2.9.0

### PR DESCRIPTION
**Pull request by pjfitzgibbons**
### Description
Release Notes for 2.9.0

- [X] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: release-notes/dashboards-observability.release-notes-2.9.0.0.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+## Version 2.9.0.0 Release Notes
+
+Compatible with OpenSearch and OpenSearch Dashboards Version 2.8.0
```
Comment (derek-ho) [inline release-notes/dashboards-observability.release-notes-2.9.0.0.md:3]: 2.9

**Review by ps48 — COMMENTED** [review]
@pjfitzgibbons we need to add all PRs that went in after 2.8 release, here in the release notes. You can follow this: https://github.com/opensearch-project/opensearch-plugins/blob/main/RELEASE_NOTES.md

File: release-notes/dashboards-observability.release-notes-2.9.0.0.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+- Integrations Plugin ([#317](https://github.com/opensearch-project/dashboards-observability/issues/317))
+
+### Bug Fixes
+- Fix Log patterns UI regressions ([#530](https://github.com/opensearch-project/dashboards-observability/issues/530))
+- [BugFix] Application Analytics - Panels not displaying availability ([#607](https://github.com/opensearch-project/dashboards-observability/pull/607))
+- Upgrade tough-cookie and semver ([#627](https://github.com/opensearch-project/dashboards-observability/pull/627))
+
+## Refactoring
+- Replace metrics panel-separator with ResizableContainer ([#602](https://github.com/opensearch-project/dashboards-observability/pull/602))
+- Remove deprecated layout editor ([#627](https://github.com/opensearch-project/dashboards-observability/pull/627))
```
Comment (ps48) [inline release-notes/dashboards-observability.release-notes-2.9.0.0.md:15]: Should the link change for this one?

**Review by ps48 — APPROVED** [review]

**Comment by ps48** [comment]
@pjfitzgibbons Can you check the DCO failure on  this commit: https://github.com/opensearch-project/dashboards-observability/pull/645/commits/9ecf1e3464e283fc40b3251882a4cabe57cc931e

**Review by derek-ho — APPROVED** [review]

**Pull request merged**